git diff --git a/plugins/tinyalsa/test/backend_conf.xml b/plugins/tinyalsa/test/backend_conf.xml
index 4520163..282bfcb 100644
--- a/plugins/tinyalsa/test/backend_conf.xml
+++ b/plugins/tinyalsa/test/backend_conf.xml
@@ -44,8 +44,8 @@
     <device name="DISPLAY_PORT-RX" rate="48000" ch="2" bits="16" />
     <device name="USB_AUDIO-RX" rate="48000" ch="2" bits="16" />
     <device name="CODEC_DMA-LPAIF_RXTX-TX-3" rate="48000" ch="1" bits="16" />
-    <device name="CODEC_DMA-LPAIF_VA-TX-0" rate="48000" ch="1" bits="16" />
-    <device name="CODEC_DMA-LPAIF_VA-TX-1" rate="48000" ch="1" bits="16" />
+    <device name="CODEC_DMA-LPAIF_VA-TX-0" rate="48000" ch="2" bits="16" />
+    <device name="CODEC_DMA-LPAIF_VA-TX-1" rate="48000" ch="2" bits="16" />
     <device name="MI2S-LPAIF_AXI-TX-PRIMARY" rate="48000" ch="1" bits="16" />
     <device name="SLIM-DEV1-TX-0" rate="48000" ch="1" bits="16" />
     <device name="USB_AUDIO-TX" rate="48000" ch="1" bits="16" />
@@ -53,6 +53,8 @@
     <device name="MI2S-LPAIF-TX-PRIMARY" rate="48000" ch="1" bits="16" />
     <device name="MI2S-LPAIF_AUD-RX-PRIMARY" rate="48000" ch="1" bits="16" />
     <device name="MI2S-LPAIF_AUD-TX-PRIMARY" rate="48000" ch="1" bits="16" />
+    <device name="MI2S-LPAIF-RX-SECONDARY" rate="48000" ch="2" bits="16" />
+    <device name="MI2S-LPAIF-TX-SECONDARY" rate="48000" ch="1" bits="16" />
     <device name="MI2S-LPAIF-RX-TERTIARY" rate="48000" ch="1" bits="16" />
     <device name="MI2S-LPAIF-TX-TERTIARY" rate="48000" ch="1" bits="16" />
     <device name="MI2S-LPAIF_RXTX-RX-PRIMARY" rate="48000" ch="1" bits="16" />

